docs/src/user_docs/assembly/io_operations.md

```diff
@@ -2,17 +2,17 @@
 Miden assembly provides a set of instructions for moving data between the stack and several other sources. These sources include:
 
 * **Program code**: values to be moved onto the stack can be hard-coded in a program's source code.
-* **Environment**: values can be moved onto the stack from environment variables. Currently, the only available environment variable is *stack_depth* which holds the current depth of the stack. In the future, other environment variables may be added.
+* **Environment**: values can be moved onto the stack from environment variables. Currently, the available environment variables are *stack_depth*, which stores the current stack depth, and *local_address*, which stores the absolute address of the local variable. In the future, other environment variables may be added.
 * **Advice tape**: values can be moved onto the stack from a non-deterministic advice tape contained within the advice provider. Values are always read from the head of the advice tape, and once a value is read, it is removed from the tape. There is no limit on the number of values in the advice tape.
 * **Memory**: values can be moved between the stack and random-access memory. The memory is word-addressable, meaning, four elements are located at each address, and we can read and write elements to/from memory in batches of four. Memory can be accessed via absolute memory references (i.e., via memory addresses) as well as via local procedure references (i.e., local index). The latter approach ensures that a procedure does not access locals of another procedure.
 
 In the future several other sources such as *storage* and *logs* may be added.
 
 ### Constant inputs
 
-| Instruction     | Stack_input | Stack_output | Notes                                      |
-| --------------- | ----------- | ------------ | ------------------------------------------ |
-| push.*a* <br> - *(1-2 cycles)* <br> push.*a*.*b* <br> push.*a*.*b*.*c*... | [...] | [a, ...] <br> [b, a, ...] <br> [c, b, a, ...] | Pushes values $a$, $b$, $c$ etc. onto the stack. Up to $16$ values can be specified. All values must be valid field elements in decimal (e.g., $123$) or hexadecimal (e.g., $0x7b$) representation. |
+| <div style="width:100px">Instruction</div> | <div style="width:100px">Stack_input</div> | <div style="width:100px">Stack_output</div> |     Notes     |
+|                   :----:                   |                   :----:                   |                    :----:                   | ------------- |
```

Comment:
I wouldn't change the title rows: these changes don't really have an effect once we compile docs with mdBook, and they make the source code more complicated.